### PR TITLE
sample-app: update docs

### PR DESCRIPTION
**Pull request by kargakis**
@bparees PTAL or assign a reviewer

### CODE REVIEW

File: examples/sample-app/README.md [hunk 1]
```diff
@@ -315,22 +349,20 @@ This section covers how to perform all the steps of building, deploying, and upd
...
+17. Make a change to your ruby sample main.html file, commit, and push it via git. If you do not have the webhook enabled, you'll have to manually trigger another build:
 
-            $ oc start-build ruby-sample-build
+        $ oc start-build ruby-sample-build
 
 
 18. Repeat step 13 (waiting for the build to complete).  Once the build is complete, refreshing your browser should show your changes.
 
-Congratulations, you've successfully deployed and updated an application on OpenShift.
+Congratulations, you've successfully deployed and updated an application on OpenShift!
```
Comment (bparees) [inline examples/sample-app/README.md]: the best delta in this PR :)

File: examples/sample-app/README.md [hunk 2]
```diff
@@ -349,69 +381,67 @@ To install the router you must know the ip address of the host the router will b
...
+
+4.  Wait for the router to start.
+
+        $ oc describe dc router --config=openshift.local.config/master/admin.kubeconfig
+        # watch for the number of deployed pods to go to 1
+
+
+5.  *Optional:* View the logs of the router.
+
+	    $ oc logs dc/router
```
Comment (bparees) [inline examples/sample-app/README.md]: extra indent?

**Comment by bparees** [comment]
one possible nit, otherwise lgtm.  I am assuming you've actually walked through the new steps to confirm it works.

File: examples/sample-app/README.md (hunk 1 above)
Comment (kargakis) [inline examples/sample-app/README.md]: I knew it

File: examples/sample-app/README.md (hunk 2 above)
Comment (kargakis) [inline examples/sample-app/README.md]: ok

**Comment by kargakis** [comment]
@bparees yes, the new steps are working as expected.

**Pull request merged**